smoke-test/src/test/java/org/opennms/smoketest/graph/GraphRestServiceIT.java

```diff
@@ -128,7 +132,65 @@ public void verifyGetContainer() {
                     .content("graphs[1].edges", Matchers.hasSize(0));
         });
     }
+    
+    @Test
+    public void verifySuggest() {
+    	createGraphMLAndWaitUntilDone();
+        given().params("s", "unknown")
+               .accept(ContentType.JSON)
+               .get("/search/suggestions/{namespace}/", "acme:regions")
+               .then().statusCode(204);
+        
+        Response response = given().params("s", "North Region")
+               .accept(ContentType.JSON)
+               .get("/search/suggestions/{namespace}/", "acme:regions");
+        LOG.info("Results from search:" + response.asString());
```

Comment:
You can use `RestAssured` for this, there is a `.log()`-statement: https://github.com/rest-assured/rest-assured/wiki/usage#request-logging

Reply:
:+1:  done.